timescaledb/how-to-guides/hyperfunctions/index.md

```diff
@@ -0,0 +1,27 @@
+# Hyperfunctions
+Hyperfunctions allows you to perform critical time-series queries quickly,
+analyze time-series data, and extract meaningful information.
+
+Some hyperfunctions are included in the default TimescaleDB product. For
+additional hyperfunctions, you need to install the
+[Timescale Toolkit][install-toolkit] PostgreSQL extension.
+
+*   [Learn about hyperfunctions][about-hyperfunctions] to understand how it works
+    before you begin using it.
+*   Install the [Toolkit extension][install-toolkit] to access more
+    hyperfunctions.
+*   Use the [approximate percentile][hyperfunctions-approximate-percentile] function.
+*   Use the [time-weighted average][hyperfunctions-time-weighted-averages] function.
+*   Use the [hyperloglog][hyperfunctions-hyperloglog] function.
```

Comment:
Statistical aggregates with stats_agg() shoudl also be here.

Reply:
Ah, I need to update this list for the function families. Good spot!